src/Evolution/Systems/ScalarAdvection/Tags.hpp

```diff
@@ -0,0 +1,31 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+namespace ScalarAdvection {
+namespace Tags {
```

Comment:
can you document this namespace too? Something like `/// \brief Tags for the ScalarAdvection system`. That makes it easier to find in the dox